code/game/objects/items/weapons/tools/shears.dm

```diff
@@ -0,0 +1,22 @@
+/obj/item/shears
+	name = "wirecutters"
+	desc = "A special pair of pliers with cutting edges. Various brackets and manipulators built into the handle allow it to repair severed wiring."
```

Comment:
> Various brackets and manipulators built into the handle allow it to repair severed wiring.

Looks suspiciously similar to wirecutter desc, forgot to adjust after copying?

Reply:
uhhh the name is also wirecutter